package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
The sibling of this file, `package-lock.json` is rather huge. It gets better with `yarn` (-50%), but it would imply more steps in the provision / setup. I guess this is just how it is, 13‘000 lines for 10 packages‽

Reply:
It eez what it eez. That file does not need to be manually touched (at least, that is what npm advertises) and git should be able to compress it very well.